cmd/nerdctl/apparmor_inspect_linux.go

```diff
@@ -19,11 +19,11 @@ package main
 import (
 	"fmt"
 
-	"github.com/containerd/nerdctl/v2/pkg/api/types"
+	"github.com/spf13/cobra"
```

Comment:
This change and all similar changes: we use a variety of styles for dependencies separation / ordering.
Uniformizing on separating non-containerd org dependencies first, then containerd dependencies.

Reply:
@apostasie  we may need ci checks to keep this consistent. We usually forget this separation/ ordering during review

Reply:
True.
Is there a tool for that or should we write something?

Reply:
this may help https://github.com/incu6us/goimports-reviser

Reply:
That tool is really neat.
Will send a PR asap.
Thanks a lot @fahedouch